lib/beaker/dsl/install_utils/puppet_utils.rb

```diff
@@ -369,15 +369,20 @@ def install_puppet_from_msi( host, opts )
           else
             dest = "C:\\Windows\\Temp\\#{host['dist']}.msi"
 
-            on host, "set PATH=\"%PATH%;#{host['puppetbindir']}\""
-            on host, "setx PATH \"%PATH%;#{host['puppetbindir']}\""
+            set_path_string = host['puppetbindir'].gsub(/"/,'')
```

Comment:
The puppetbindir is added to the env during configuration (in host_prebuilt_steps::set_env) so I think that this secondary setting is unnecessary.